### DISCUSSION TITLE
Localization files

### DISCUSSION THREAD

**Discussion by iamsubingyawali** [Kinnara/ModernWpf#423] [General]
I am using this library for my WPF. While building my app, many language specific folders are generated automatically as shown below:

![image](https://user-images.githubusercontent.com/45819206/149623430-a9460c81-630e-452e-bff7-80773abfa258.png)

Inside each of these folder, there are two files as below:

![image](https://user-images.githubusercontent.com/45819206/149623457-6941decb-b58b-4258-a52e-2e13efac5f8e.png)

How do I get rid of these?

**Comment by alxkalinin** [comment]
Add the following property in your .csproj file
```xml
    <PropertyGroup>
        ...
        <SatelliteResourceLanguages>en</SatelliteResourceLanguages>
    </PropertyGroup>
```

**Comment by iamsubingyawali** [comment]
Thanks!
